tests/unit/mock/config/compliance/compliance/nokia_sros/sros_basic_feature.py

```diff
@@ -0,0 +1,4 @@
+features = [
+    {"name": "System Configuration", "ordered": False, "section": ["System Configuration"]},
```

Comment:
Just making sure that this is the "right" level of parent that we have. I am not as familiar with sros configs, but in ios and ios-like configs it's reasonable to have a single parent such as "router bgp" be a parent, do we have the "right" level of parent here?

Reply:
I'm not a SrOS expert either but when it was discussed with the client they stated that it made the most sense to use the banners as section delimiters/parents due to the structure of the configuration. For example, if you ignore the section banners then it becomes quite difficult to try and define the parent line for a section as there is quite a bit of duplication throughout the config, ie `system` is at the same level in multiple sections and you need to potentially look at the next line and indentation to determine the differing section. If someone has a better idea of how to handle it I'm welcome to suggestions.

Reply:
@pszulczewski you have any thoughts?

Reply:
SROS config structure relies on indentation, delimiting contexts by 4 spaces, similarly to python. [You can even use python tokenizer to parse out sros onto bits, tested! ;-) ]
There is one parent `system` and other contexts like `system security` are children, in this case `security` is a child of parent `system` parent.
Banners are separating some contexts (child contexts in this case) only for the convenience of users, system doesn't really care when you deploy configuration.

In this case when you develop config compliance it makes sense to keep these sections to have better granularity for features. In this case you can treat separately `system` as System General, `system security` as Security and so on.

This looks to me like the right approach for config compliance.

Reply:
Thanks for your confirmation that I'm on the right track @pszulczewski!